src/main/java/edu/umich/ctools/sectionsUtilityTool/FriendServlet.java

```diff
@@ -23,16 +22,18 @@
 	private static final String INVITE_EMAIL = "inviteEmail";
 	private static final String INSTRUCTOR_EMAIL = "instructorEmail";
 	private static final String INSTRUCTOR_NAME = "instructorName";
+	private static final String SEND_EMAIL = "sendEmail";
 	private static final String PARAMETER_ID = "id";
 	private static final String PARAMETER_INSTRUCTOR_EMAIL = "inst_email";
 	private static final String PARAMETER_INSTRUCTOR_FIRST_NAME = "inst_first_name";
 	private static final String PARAMETER_INSTRUCTOR_LAST_NAME = "inst_last_name";
-	
+	private static final String PARAMETER_SEND_EMAIL = "send_email";
```

Comment:
Doesn't it always send at least 1 email?  The name should clearly indicate what will happen.  E.g. "notify_instructor", or "send_instructor_email" or whatever is true.